README.md

```diff
@@ -7,10 +7,64 @@ This workflow does the primary data processing for RNAseq data: alignment, QC, r
+| num_reads                  | The number of reads falling into the gene region, in a non-strand specific counting mode                                                                                                                                                                                                                                                                                                                               |
+| num_reads_fw               | The number(s) of reads falling into the gene region, in a strand specific counting modes. Applying the featureCounts option for standedness either 1/forward (_fw) or 2/reverse (_rv). Which option is the appropriate one for strandedly sequenced data is dependent on the library preparation step. For example, the Illumina TruSeq stranded RNA kit usually produces data that fits the option 2/reverse (_rv).   |
+| num_reads_rv               |                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| FPKM_no_mt_rrna_trna_chrxy                       | Unstranded FPKM for which the count values for rRNA, tRNA, chrX, chrY and Mt were omitted during the normalization process. (For example for FPKM these counts are subtracted from the sum of all counts).                                                                                                                                                                                                             |
+| FPKM_no_mt_rrna_trna_chrxy_fw                    | FPKM like before but just forward/sense strand.                                                                                                                                                                                                                                                                                                                                                                        |
+| FPKM_no_mt_rrna_trna_chrxy_rv                    | FPKM like before but just reverse/antisense stand                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
In addition: The terms "sense" and "antisense" are inappropriate here and should be omitted. featureCounts does not count only reads mapping to the sense/antisense strand. What it really does is take the mapping strand as-is (`_fw`) or reverse it (`_rv`). In other words, in `_rv` mode, reads mapping to the + strand are considered to map to the - strand. Whether this strand is sense or antisense is irrelevant.

In fact, even "forward" and "reverse" are a bit of a misnomer and can be confusing, because strictly speaking the meaning of the columns is not really related to the forward/reverse strand. Better names would be `_stranded` and `_reverse_stranded`. I'm not suggesting to rename the columns. I guess this is what we have to stick with for historical reasons. I am just trying to explain myself, because I know from countless discussion how confusing this subject is.

Reply:
> sense/antisense

You mean because the expression data itself, dependent on the lab protocol, may yield reads in sense or antisense direction, or not? Could the workflow itself actually deal with such data?

Maybe "forward-mapping strand" and "reverse-mapping strand" would fit better then.

Reply:
The current description "but just reverse/antisense strand" sounds like only the reads on the reverse strand or the antisense strand are counted. Firstly, the reverse strand and the antisense strand are not the same thing. Secondly, this is not the real meaning of these columns. What featureCounts really does in the `_rv` column is to *invert* the meaning of the `BAM_FREVERSE` flag. That is, when the `BAM_FREVERSE` flag is set, the RNA molecule is assumed to be transcribed from the forward strand, and vice versa.

The description of the column `num_reads_rv` is better IMO. Why not refer to it?

```
| FPKM_no_mt_rrna_trna_chrxy_fw                    | FPKM like before but in strand-specific counting mode analogous to num_reads_fw |
| FPKM_no_mt_rrna_trna_chrxy_rv                    | FPKM like before but in reversed strand-specific counting mode analogous to num_reads_rv |
```

Reply:
O.k., I guess we have a different understandings of some basic concept/wording here. What I understand is that a forward match is a alignment match in the direction of the annotated feature. Certainly not that forward/reverse refers to Watson/Crick strands -- just to make sure. And, I already outlined before how I think this basic assumption, together with how the annotation (on which strand features are annotated) looks like and what data the RNA-seq protocol produces (the read direction) seem to warrant the usage of the terms "sense" and "antisense".

If this is wrong, or if you have a correcter or more precise description, it would be really great if you could contribute them.

Reply:
O.k., these values are produced by `featureCounts_2_FpkmTpm`, which in turn gets the values from the `.s{0,1,2}` files. These files are from calls to `featureCounts -s {0,1,2}`. 

According to the documentation, the `-s` option determines whether `featureCounts` does unstranded (0), stranded (1), or reversely stranded (2) counting. For paired-end reads the strand of the first read (r1) is decisive. 

AFAIR, in *most* protocols, the first read has the sequence of the transcript, but that might not be true for all protocols. So it might be true that 1 and 2 cannot simply be interpreted as forward/sense and reverse/antisense, because it could in principle also be the other way around. (I guess a protocol that always produces Watson reads (or Crick) is about impossible!)

So better names would be "_strand", "_unstranded" and "_reverse_stranded", just reiterating the names used in the `featureCounts` docs.

@NagaComBio Can we resolve this? If we don't update the column names, we at least need to change the documentation.

Reply:
@vinjana We are used to `_fw` and `_rv` and in downstream scripts, so we could leave it and update the README about the column names.